src/themes/eLife/styles.css

```diff
@@ -300,3 +310,93 @@ blockquote {
     line-height: 1.56522;
   }
 }
+
+:--cite-group {
```

Comment:
Now that you have started implementing this, I'd be interested in your thoughts as to whether it's a good approach or not. As a CSS developer does it feel OK to work with?